website/docs/docs/user-interface/can-i-deploy.md

```diff
@@ -3,32 +3,32 @@ title: Can I Deploy
 ---
 
 ## What is Can I Deploy 
-Can I Deploy page is the web version of our popular CLI tool [can-i-deploy](https://docs.pact.io/pact_broker/can_i_deploy), that queries the "Matrix". This UI provides a rich query interface for the Matrix to ensure you can safely deploy your application, providing additional context not readily available during CLI usage.
+Can I Deploy page is the web version of our popular CLI tool [can-i-deploy](https://docs.pact.io/pact_broker/can_i_deploy), which queries the "Matrix". This UI provides a rich query interface for the Matrix to ensure you can safely deploy your application. It provides additional context not readily available during CLI usage.
 
-You can find `Can I Deploy` button from the top navigation bar on overview page.
+You can find `Can I Deploy` button in the top navigation bar on the overview page.
 
 ## Walkthrough
 <iframe width="560" height="315" src="https://www.youtube.com/embed/e-l21IjI_d8" title="Can I Deploy introduction" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 
 ## Can I Deploy layout
 
-The Can I Deploy page contains three main areas.
+The Can I Deploy page contains three main areas:
 
 &nbsp;
 
 ![can i deploy screenshot](/ui/cid.png)
 
 &nbsp;
 
-**1 -**
-UI selectors for performing a Matrix query, where you can choose your application from the application list, the application version and the deployment target. For more information, see the [Selectors](#Selectors) section.
+**UI selectors**
+It helps you choose your application from the application list, the application version and the deployment target. For more information, see the [Selectors](#Selectors) section.
 
-**2 -**
-A list of messages summarising if you are safe to deploy your chosen application version. 
+**Messaging list**
+It summarizes if you are safe to deploy your chosen application version. 
```

Comment:
Are we moving to American spelling? 😞 

I may not have noticed this elsewhere.

Reply:
We are trying to bring consistency to the doc set. But if you need it like it was earlier, I can edit the spellings.